### DISCUSSION TITLE
OneBot Connect WebSocket 更名

### DISCUSSION THREAD

**Discussion by abrahum** [howmanybots/onebot#246] [想法 / 建议]
提议：

将 `正向WebSocket` 更名为 `协议端WebSocket`
将 `反向WebSocket` 更名为 `应用端WebSocket`

理由：

正向与反向存在主观的主从正负区别，无法见文明义确定WebSocket的方向；
更名后只需知晓名字中的为服务器端，即可快速分辨连接的方向。

**Comment by crazywhalecc** [comment]
持保留意见。另外，协议端和应用端这两个名称也是一个纠结的问题。现在标准中统称协议端为 `实现端`，目前还没有人提出有关这部分的想法和建议。

**Comment by ichizi** [comment]
正向和逆向的概念是有一定的理解成本，回想刚接触的时候，什么是正向 什么是逆向 理解了半天。

**Comment by sgpublic** [comment]
同意，最近刚开始研究，正向和逆向究竟是谁连谁我看了半天文档没理解到，到头来还是看已有实现的代码才看出来的

**Comment by stdrc** [comment]
感觉可以 WebSocket A2I（App-to-Implementation）和 WebSocket I2A
